src/task6/MyArrayList.java

```diff
@@ -0,0 +1,56 @@
+package task6;
+
+import java.util.Arrays;
+
+public class MyArrayList<T> {
+    private Object[] list;
+    private int length;
```

Comment:
I think it should be static

Reply:
But if variable "length" would be static - then all objects of type MyArrayList would be same variable "length"